src/readqps.jl

```diff
@@ -231,44 +271,69 @@ function read_rhs_section(qps, objname, connames, contypes)
         end
         is_section_header(line) && break  # reached the end of this section
 
+        rhs = strip(line[5:12])
+        if !rhs_name_read
+            # Record this as the RHS
+            rhsname = rhs
+            rhs_name_read = true
+        elseif rhsname != rhs
+            # Rim RHS, ignore this line
```

Comment:
I thought we said we would use logging to warn users of such inconsistencies?!

Reply:
I was waiting for a first round of comments before adding the logging part :)